src/main/java/Main.java

```diff
@@ -0,0 +1,29 @@
+import dao.FruitDao;
+import dao.impl.FruitDaoImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import service.TransactionExecutor;
+import service.TransactionParser;
+import service.impl.ReportServiceImpl;
+import service.impl.TransactionExecutorImpl;
+import service.impl.TransactionParserImpl;
+
+public class Main {
+    private static final String INPUT_CSV_PATH = "src/main/resources/input.csv";
+    private static final String REPORT_CSV_PATH = "src/main/resources/report.csv";
+
+    public static void main(String[] args) {
+        FruitDao fruitDao = new FruitDaoImpl();
+        TransactionParser transactionParser = new TransactionParserImpl();
+        TransactionExecutor transactionExecutor = new TransactionExecutorImpl();
+        try {
+            transactionParser.parse(Files.readAllLines(Path.of(INPUT_CSV_PATH)))
```

Comment:
We should move logic or reading/writing files in separate services

Reply:
Another mentor says like "There is Files.readAllLines() in java core and I do not think we should reinvent the wheel creating our custom interface.". So i deleted Writer and Reader services.